### DISCUSSION TITLE
Rendering content in Next.js App Router

### DISCUSSION THREAD

**Discussion by noahflk** [sdorra/content-collections#13] [Q&A] [answered]
Just discovered your project and really liking it so far. One question: what's the go-to way to render the post content `post.body` in Next.js App Router?

**Comment by sdorra** [comment]
It depends on your content. Content-collections returns just a string for the content of your files. If your content is markdown you can use something like [react-markdown](https://github.com/remarkjs/react-markdown) or you can use the `transform` function of content-collections to turn your markdown into html at build time. Can you describe your files and what's your goal?

**Reply by sdorra** [reply, marked as answer]
There is a new package which simplifies the handling of markdown:

- [@content-collections/markdown](https://www.content-collections.dev/docs/samples/markdown)

And another one which simplifies the usage of mdx:

- [@content-collections/mdx](https://www.content-collections.dev/docs/samples/mdx)
